### ISSUE TITLE
Test suite is time zone sensitive

### ISSUE THREAD

**Issue by rycee** [pytrainer/pytrainer#100]
I'm working on packaging version 1.11.0 for the [NixOS](https://nixos.org) GNU/Linux distribution and discovered the following issue in the test suite:

```
======================================================================
FAIL: test_getDateTime (pytrainer.test.lib.test_date.DateFunctionTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/nix-build-pytrainer-1.11.0.drv-0/pytrainer-v1.11.0-src/pytrainer/test/lib/test_date.py", line 48, in test_getDateTime
    self.assertEqual(datetime.datetime(2015, 11, 24, 19, 29, 5, tzinfo=tzlocal()), localtime)
AssertionError: datetime.datetime(2015, 11, 24, 19, 29, 5, tzinfo=tzlocal()) != datetime.datetime(2015, 11, 24, 17, 29, 5, tzinfo=tzlocal())

======================================================================
FAIL: test_workout_summary (pytrainer.test.imports.test_garmintcxv2.GarminTCXv2Test)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/nix-build-pytrainer-1.11.0.drv-0/pytrainer-v1.11.0-src/pytrainer/test/imports/test_garmintcxv2.py", line 34, in test_workout_summary
    self.assertEquals(summary, garmin_tcxv2.activitiesSummary)
AssertionError: Lists differ: [(0, False, '2012-10-14T12:02:... != [(0, False, '2012-10-14T10:02:...

First differing element 0:
(0, False, '2012-10-14T12:02:42', '10.12', '00:39:51', 'Running')
(0, False, '2012-10-14T10:02:42', '10.12', '00:39:51', 'Running')

- [(0, False, '2012-10-14T12:02:42', '10.12', '00:39:51', 'Running')]
?                          ^

+ [(0, False, '2012-10-14T10:02:42', '10.12', '00:39:51', 'Running')]
?                          ^


======================================================================
FAIL: test_activity_starttime (pytrainer.test.core.test_activity.ActivityTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/nix-build-pytrainer-1.11.0.drv-0/pytrainer-v1.11.0-src/pytrainer/test/core/test_activity.py", line 72, in test_activity_starttime
    self.assertEquals(self.activity.starttime, '12:58:23 PM')
AssertionError: '12:58:23' != '12:58:23 PM'
```

Our builds are done in the UTC timezone and C locale for maximum reproducibility. Unfortunately from the above test errors it appears the pytrainer test suite needs some specific timezone and locale. Would it be possible to change the test suite to be agnostic w.r.t timezone and locale? Alternatively, which timezone and locale does the test suite expect?

**Comment by viiru-** [comment]
The timezone test tests for the conversion between UTC and local time, and thus having TZ set to UTC will not work for it. The test (created by David) required UTC+3, so I looked up a suitable timezone that doesn't have DST (so that the test isn't dependent on the date). The one I found was Europe/Kaliningrad, and it's set for travis in .travis.yml (TZ=Europe/Kaliningrad).

For LC_TIME we could just as well default to C, but travis happens to default to LC_TIME=en_US.UTF-8, so that's what we currently have.
